uber/model_checks.py

```diff
@@ -148,8 +149,12 @@ def email(attendee):
     if len(attendee.email) > 255:
         return 'Email addresses cannot be longer than 255 characters.'
 
-    if (c.AT_OR_POST_CON and attendee.email and not re.match(c.EMAIL_RE, attendee.email)) or (not c.AT_OR_POST_CON and not re.match(c.EMAIL_RE, attendee.email)):
-        return 'Enter a valid email address'
+    if (c.AT_OR_POST_CON and attendee.email) or not c.AT_OR_POST_CON:
+        try:
+            validate_email(attendee.email)
+        except EmailNotValidError as e:
```

Comment:
@khyperia and I went through this and it will throw an EmailNotValidError for an empty string which tells people they need at least one `@` sign, so this should work great.